examples/flutter_firebaseauth/lib/home_page.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_firebaseauth/authentication/authentication.dart';
+
+class HomePage extends StatelessWidget {
+  final AuthenticationAuthenticated userState;
+
+  const HomePage({Key key, this.userState}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    final AuthenticationBloc authenticationBloc =
+        BlocProvider.of<AuthenticationBloc>(context);
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Home'),
+      ),
+      body: Container(
+        child: Center(
+            child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text(
+              'Welcome,\n' + this.userState.user.displayName,
```

Comment:
minor but we can use interpolation here:

```dart
Text('Welcome,\n${userState.user.displayName}')
```

Reply:
Done.